src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -583,6 +628,19 @@ def load_arguments(self, _):
         c.argument('enable_network_observability', action='store_true', is_preview=True, help="enable network observability for cluster")
         c.argument('enable_cost_analysis', is_preview=True, action='store_true')
         c.argument('disable_cost_analysis', is_preview=True, action='store_true')
+        # azure container storage
+        c.argument('enable_azure_container_storage', arg_type=get_enum_type(storage_pool_types),
+                   help='enable azure container storage and define storage pool type')
+        c.argument('disable_azure_container_storage', action='store_true',
+                   help='Flag to disable azure container storage')
```

Comment:
Is it better to rename the param `--enable-azure-container-storage` as `--enable-xxx` seems to be a boolean type of param and may confuse constomers.

Reply:
This is as per the spec designed and approved by the AKS team. Shared the spec with you offline.